[2023-11-26 17:53:36,326] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_covid_etl.create_fact_district_daily manual__2023-11-26T17:53:35.308758+00:00 [queued]>
[2023-11-26 17:53:36,340] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_covid_etl.create_fact_district_daily manual__2023-11-26T17:53:35.308758+00:00 [queued]>
[2023-11-26 17:53:36,341] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-11-26 17:53:36,341] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-11-26 17:53:36,341] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-11-26 17:53:36,358] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_fact_district_daily> on 2023-11-26 17:53:35.308758+00:00
[2023-11-26 17:53:36,365] {standard_task_runner.py:52} INFO - Started process 527 to run task
[2023-11-26 17:53:36,371] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_covid_etl', 'create_fact_district_daily', 'manual__2023-11-26T17:53:35.308758+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/get_data.py', '--cfg-path', '/tmp/tmp25gnygv9', '--error-file', '/tmp/tmpg52jykie']
[2023-11-26 17:53:36,372] {standard_task_runner.py:80} INFO - Job 84: Subtask create_fact_district_daily
[2023-11-26 17:53:36,449] {task_command.py:369} INFO - Running <TaskInstance: data_covid_etl.create_fact_district_daily manual__2023-11-26T17:53:35.308758+00:00 [running]> on host 79a606f4be36
[2023-11-26 17:53:36,546] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=oce
AIRFLOW_CTX_DAG_ID=data_covid_etl
AIRFLOW_CTX_TASK_ID=create_fact_district_daily
AIRFLOW_CTX_EXECUTION_DATE=2023-11-26T17:53:35.308758+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-26T17:53:35.308758+00:00
[2023-11-26 17:53:36,547] {python.py:173} INFO - Done. Returned value was: None
[2023-11-26 17:53:36,559] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_covid_etl, task_id=create_fact_district_daily, execution_date=20231126T175335, start_date=20231126T175336, end_date=20231126T175336
[2023-11-26 17:53:36,583] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-26 17:53:36,615] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
